core-bundle/src/Util/SymlinkUtil.php

```diff
@@ -26,20 +27,15 @@ public static function symlink(string $target, string $link, string $rootDir): v
     {
         static::validateSymlink($target, $link, $rootDir);
 
-        $fs = new Filesystem();
-
-        if (!$fs->isAbsolutePath($target)) {
-            $target = $rootDir.'/'.$target;
-        }
+        $target = Path::makeAbsolute($target, $rootDir);
```

Comment:
What if `$target` is an absolute path already?

Reply:
Than it's returned unaltered.